### DISCUSSION TITLE
what is {motion}?

### DISCUSSION THREAD

**Discussion by CunningMonkey** [kylechui/nvim-surround#252] [Q&A] [answered]
What does "motion" in the documentation refer to? Does it refer to the "motion" in Vim, or is it a term you have defined yourself? If it is something you have defined, where can I find the specific definition, for example, for "iw"?

**Comment by kylechui** [comment, marked as answer]
It refers to any builtin-motion (e.g. `iw, 2i", 4j, ...`) or any motion defined by other plugins. It's defined in `:h motion`.

**Comment by CunningMonkey** [comment]
cool! thank you for the awesome tool!
